### PR TITLE
My enumerable

### PR DESCRIPTION
**Pull request by Wes-Isaac**
# created List class and methods

In this pull request I did the following:

- Create myEnumerable method.
- implemented methods all? any? and filter.

### CODE REVIEW

File: my_list.rb [hunk 1]
```diff
@@ -0,0 +1,16 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*list)
+    @list = list
+  end
+end
```
Comment (hiromataba) [inline my_list.rb]: - [ ] Good job while implementing this requirement of having a MyLIst class :+1: , however, there is something that is missing. Please check this requirement over here below:
list

File: README.md [hunk 2]
```diff
@@ -1,72 +1,52 @@
-![](https://img.shields.io/badge/Microverse-blueviolet)
-
-# Project Name
-
-> Description the project.
-
+# Morse Code Decoder
```
Comment (hiromataba) [inline README.md]: - [x] Kindly change the title of your project to the corresponding title of the project just to make your project more professional :smile_cat:

**Review by hiromataba — CHANGES_REQUESTED** [review]
Hi :wave:  @Wes-Isaac  and  @KDlamini , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ZeenatLawal — APPROVED** [review]
Hi @Wes-Isaac and @KDlamini, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

## Congratulations! 🎉 
6vEo


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Pull request merged**